aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/SubscriptionProcessor.java

```diff
@@ -115,6 +115,14 @@ void startSubscriptions() {
             ));
             latch.await(SUBSCRIPTION_START_TIMEOUT_MS, TimeUnit.MILLISECONDS);
         })
+        .doOnError(exception -> {
+            LOG.warn("An error occurred with one of the subscriptions to the remote data store for " +
```

Comment:
super minor nit: "datastore" vs. "data store"

Reply:
Actually, neither!

It's the _`DataStore`_.